ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/benchmark/ScrollActivity.kt

```diff
@@ -95,9 +94,12 @@ fun SomeScreenContent() {
                     Text("Item $item")
                 },
                 toggleControl = {
-                    Icon(
-                        imageVector = ToggleChipDefaults.switchIcon(checked = itemChecked),
-                        contentDescription = "Item $item ${if (itemChecked) "Checked" else "Unchecked"}"
+                    Switch(
+                        checked = itemChecked,
+                        modifier = Modifier.semantics {
+                            this.contentDescription =
+                                "Item $item ${if (itemChecked) "Checked" else "Unchecked"}"
```

Comment:
From the samples, it looks like the suggestion is to just use "Checked" and the context of the label still applies.

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:wear/compose/compose-material/samples/src/main/java/androidx/wear/compose/material/samples/ToggleChipSample.kt?q=ToggleChip

Should we check with which it is? or just remove "Item $item " here?

Reply:
I'm leaning for latter, to remove "Item $item " - going to update now